MCprep_addon/materials/prep.py

```diff
@@ -150,6 +155,12 @@ def draw_mats_common(self, context):
 	row = self.layout.row()
 	row.prop(self, "optimizeScene")
 
+	# EEVEE won't benefit from this anyway, so best to disable it
```

Comment:
I would argue against this, as having emission means it at least appears brighter even if it doesn't actually give off light. Plus, people can technically use light probes. What harm is there in having emission on in Eevee? I also find that I often switch from eevee to cycles when I first get to rendering as I wasn't thinking about it right away.

<img width="405" alt="emission evidence" src="https://user-images.githubusercontent.com/2958461/223741669-5634b4de-257f-44b5-8409-8f48cc89d41e.png">